Gems/AtomLyIntegration/EditorModeFeedback/Code/Source/EditorModeFeedbackFeatureProcessor.h

```diff
@@ -22,6 +23,7 @@ namespace AZ
         //! Feature processor for Editor Mode Feedback visual effect system.
         class EditorModeFeatureProcessor
             : public RPI::FeatureProcessor
+            , public AZ::TickBus::Handler
```

Comment:
update: Would be better to make this private and move the implementation to the private section of the class too

Reply:
fixed 👍